op-challenger/game/fault/responder/responder.go

```diff
@@ -95,7 +95,10 @@ func (r *FaultResponder) PerformAction(ctx context.Context, action types.Action)
 		// Always upload local preimages
 		if !preimageExists {
 			err := r.uploader.UploadPreimage(ctx, uint64(action.ParentIdx), action.OracleData)
-			if err != nil {
+			if err == preimages.ErrChallengePeriodNotOver {
+				r.log.Info("Large Preimage Squeeze failed, challenge period not over")
```

Comment:
We're going to log this every block and it's an expected situation so it probably should just be debug level.